src/plugins/bfetch/server/plugin.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { CoreStart, PluginInitializerContext, CoreSetup, Plugin } from 'src/core/server';
+import { schema } from '@kbn/config-schema';
+import { StreamingResponseHandler, removeLeadingSlash } from '../common';
+import { createStreamingResponseStream } from './streaming';
+
+// eslint-disable-next-line
+export interface BfetchServerSetupDependencies {
+  
```

Comment:
Nit: This signature has newlines and empty space, but the one on line 31 doesn't. Might want to be consistent, stylistically.

Reply:
@chrisdavies I removed the blank space.

@lizozom I wouldn't want to remove those interface for consistency and when somebody needs to add a dependency they have a clear place where to do it. Also, if somebody wants to instantiate the plugin they have an exported type that specifies dependencies of this plugin.